### PR TITLE
Embed Ask form within App container

### PR DESCRIPTION
**Pull request by kadamwhite**
This adds a new React component that is capable of injecting the Ask form's embed script URL into the document in order to render the form into a style-able container within the visualization application.

Component state is managed entirely through direct DOM interaction, since we cannot re-render the component without deleting the script-injected form. This component should never be allowed to update/re-render.

embedded-form

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,6 +14,7 @@ To get started quickly with local development, just save this JSON as `config.js
 {
   "jsonURI": "http://localhost:3000/data/",
   "formId": "5816b369d489c974e0cc8be8",
+  "formScript": 'http://localhost:3000/data/5816b369d489c974e0cc8be8.js',
```
Comment (iros) [inline README.md]: Double quotes?

File: src/components/EmbeddedAskForm.js [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+    // Never re-render, it would clobber the embedded form!
+    return false;
+  }
+
+  injectScript() {
+    const { formScript } = this.props;
+
+    // Inspired by Google Analytics' script injection code:
+    // Use a pre-existing script tag as the model for the one we will insert
+    const scriptTag = document.getElementsByTagName('script')[0];
```
Comment (iros) [inline src/components/EmbeddedAskForm.js:24]: Should this select an element by classname or id? this feels a bit brittle if someone goes and changes the order of things in the index.html file.

File: src/components/EmbeddedAskForm.js [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+          className="embedded-form"
+          ref={(node) => { this.askForm = node; }}
+        />
+        <button
+          className="form-toggle-button"
+          type="button"
+          onClick={() => this.toggleForm()}
+          ref={(node) => { this.button = node; }}
+        >
+          Tell the president-elect what you think
```
Comment (iros) [inline src/components/EmbeddedAskForm.js:88]: This needs to be template text

**Review by iros — APPROVED** [review]
Left a few comments. Some quick fixes, I think.

File: src/components/EmbeddedAskForm.js (hunk 2 above)
Comment (kadamwhite) [reply src/components/EmbeddedAskForm.js:24]: this just grabs the first script on the page; what it is is unimportant, only the `script`-ness matters. I hadn't seen the trick before but it seems reliable.

File: src/components/EmbeddedAskForm.js (hunk 3 above)
Comment (kadamwhite) [reply src/components/EmbeddedAskForm.js:88]: agreed but i want to do that as a follow-up

File: README.md (hunk 1 above)
Comment (kadamwhite) [reply README.md]: ayup! fixed.

**Pull request merged**